### ISSUE TITLE
Any Gcode command after G28 does not work at all

### ISSUE THREAD

**Issue by 3dlpm** [Stephan3/dwc2-for-klipper#86]
Any command sent to Klipper via DWC2 does not work except M112 and G28 and G90.
G28 correctly executes and every axis moves correctly and homes. 
Every command executed after that, yields a dictionary _params error:
G0 X10 Y10 Z10 F500
failed: G0'dict' object has no attribute '_params'

Klipper commands also do not work. Example: QUERY_ENDSTOPS returns:

QUERY_ENDSTOPS
failed: QUERY_ENDSTOPS'dict' object has no attribute 'get_commandline'

Basically it is not useable at all. 
Followed installation and configuration instructions exactly, except for adjustments for DWC2 web stuff as the files in the instructions are no longer there (renamed).

Thanks for the help,

**Comment by pluuuk** [comment]
This is because Klipper updated their gcode.py API and Stefan hasn't reviewed PR #75 , please clone this repo and give it a try: https://github.com/pluuuk/dwc2-for-klipper.git
